### PR TITLE
fix: `ScheduleSettings.test.tsx` failing due to daylight savings time ending

### PR DESCRIPTION
**Pull request by bnussman-akamai**
## Description 📝
- Fixes `ScheduleSettings.test.tsx`  that broken due to Daylight savings time ending

## Changes  🔄
- Makes the unit test use `utc` for asserting the time
- Adds a DST-compatible unit test for `America/New_York`

### Verification steps 
- `yarn test ScheduleSettings.test.tsx`
- Or verify Github Actions passes

## As an Author I have considered 🤔

- [x] 👀 Doing a self review
- [x] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [x] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [x] 🧪 Providing/Improving test coverage
- [x] 🔐 Removing all sensitive information from the code and PR description
- [x] 🚩 Using a feature flag to protect the release
- [x] 👣 Providing comprehensive reproduction steps
- [x] 📑 Providing or updating our documentation
- [x] 🕛 Scheduling a pair reviewing session
- [x] 📱 Providing mobile support
- [x] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/src/features/Linodes/LinodesDetail/LinodeBackup/ScheduleSettings.test.tsx [hunk 1]
```diff
@@ -56,9 +92,15 @@ describe('ScheduleSettings', () => {
...
-    await findByText('Monday');
-    await findByText('00:00 - 02:00');
+    await findByText('Wednesday');
+
+    // Because of Daylight savings time, offset will be -4 or -5.
+    const offset = DateTime.now().setZone('America/New_York').offset / 60;
+
+    // W10 indicates that your backups should be taken between 10:00 and 12:00 UTC.
+    // W10 in America/New_York will be 06:00 - 08:00 EDT (UTC-5) or 05:00 - 07:00 EST (UTC-4).
+    await findByText(`0${10 + offset}:00 - 0${10 + 2 + offset}:00`);
```
Comment (bnussman-akamai) [inline packages/manager/src/features/Linodes/LinodesDetail/LinodeBackup/ScheduleSettings.test.tsx:102]: I don't love this but I couldn't find a clean way to mock EST vs EDT at the `/v4/profile` level

File: packages/manager/src/features/Linodes/LinodesDetail/LinodeBackup/ScheduleSettings.test.tsx [hunk 2]
```diff
@@ -56,9 +92,15 @@ describe('ScheduleSettings', () => {
...
-    await findByText('Monday');
-    await findByText('00:00 - 02:00');
+    await findByText('Wednesday');
+
+    // Because of Daylight savings time, offset will be -4 or -5.
+    const offset = DateTime.now().setZone('America/New_York').offset / 60;
+
+    // W10 indicates that your backups should be taken between 10:00 and 12:00 UTC.
+    // W10 in America/New_York will be 06:00 - 08:00 EDT (UTC-4) or 05:00 - 07:00 EST (UTC-5).
+    await findByText(`0${10 + offset}:00 - 0${10 + 2 + offset}:00`);
```
Comment (bnussman-akamai) [inline packages/manager/src/features/Linodes/LinodesDetail/LinodeBackup/ScheduleSettings.test.tsx:102]: I couldn't find a way to mock DST/no DST via `/v4/profile` but I'm sure there is another way to do it. 

I don't love this solution but it should work. Open to ideas here.

**Review by mjac0bs — APPROVED** [review]
All passing:
image

Thanks for the fix and the clear comments to document, as well! This looks clean now. 

Down with Daylight Savings. 👎🏼

**Review by jdamore-linode — APPROVED** [review]
Nice, thanks @bnussman-akamai!

**Pull request merged**